packages/@aws-cdk/aws-lambda-nodejs/lib/function.ts

```diff
@@ -104,6 +118,7 @@ export class NodejsFunction extends lambda.Function {
       cacheDir: props.cacheDir,
       nodeVersion: extractVersion(runtime),
       nodeDockerTag: props.nodeDockerTag || `${process.versions.node}-alpine`,
+      projectRoot: path.dirname(path.resolve(projectRoot)),
```

Comment:
Why dirname?

Reply:
This is wrong... was because `findGitPath()` returned the path with `.git`.